### DISCUSSION TITLE
Custom Type Keywords

### DISCUSSION THREAD

**Discussion by Gekocaretaker1** [dotnet/csharplang#5246] [Language Ideas] [answered]
Something that I have been trying to find is a way to have custom type keywords.
There is a workaround _Using Alias_ but it is per file.
I was thinking that classes that have been set up correctly could have a keyword set to it.
This could be used for really long class names, or for creating custom types.

**Comment by CyrusNajmabadi** [comment, marked as answer]
Using aliases can be global for the project with `global using X = ...`

**Reply by Gekocaretaker1** [reply]
This works perfectly, thanks!
If only I knew .net 6 came out sooner.
